mne/viz/utils.py

```diff
@@ -1929,6 +1948,59 @@ def _annotation_radio_clicked(label, radio, selector):
     selector.rectprops.update(dict(facecolor=color))
 
 
+def _setup_butterfly(params):
+    """Set butterfly view of raw plotter."""
+    if 'ica' in params:
+        return
+    butterfly = not params['butterfly']
+    ax = params['ax']
+
+    if butterfly:
+        from ..selection import _SELECTIONS
+        from .raw import _setup_browser_selection
```

Comment:
Yeah, it's for dealing with circular imports. I can move it at the start of the function too.